app/assets/javascripts/code/services/_fixed_sessions.js

```diff
@@ -144,7 +144,7 @@ export const fixedSessions = (
       const clusteredSessions = clusterer(sessionsToCluster, map);
       const lonelySessions = sessions.filter(isNotIn(clusteredSessions));
 
-      sessionsUtils.updateCrowdMapLayer(sessionsIds(clusteredSessions));
+      sessionsUtils.updateCrowdMapLayer(sessionsIds(clusteredSessions), "Fixed");
```

Comment:
You could create a Constants.js file and export the constant from there. That way you are sure it has only one single point of truth.